src/Magnum/ImGuiIntegration/Test/ContextGLTest.cpp

```diff
@@ -36,18 +36,26 @@
 
 namespace Magnum { namespace ImGuiIntegration { namespace Test { namespace {
 
-enum class Modifier: Int {
-    Shift, Ctrl, Alt, Super
+struct InputEvent {
+    enum class Modifier: Int {
+        Shift = 1 << 0,
+        Ctrl = 1 << 1,
+        Alt = 1 << 2,
+        Super = 1 << 3
+    };
+    typedef Containers::EnumSet<Modifier> Modifiers;
```

Comment:
You could move the `_modifiers` and `modifiers()` here as well, to deduplicate the code a bit.